packages/sdk/src/clients/optimisticOracle/client.e2e.ts

```diff
@@ -0,0 +1,20 @@
+require("dotenv").config();
```

Comment:
Just out of curiosity, what's the rationale for unit tests vs end-to-end tests that aren't run in ci? Is there a process for determining what code needs which in the sdk?

Reply:
well the rationale is that we dont have hardhat setup in the sdk, so its just easier to run the tests against a real data source.  ideally id like to turn these all into unit tests, and its in the backlog for now.